kotlin-react-dom/src/main/generated/react/dom/PointerType.kt

```diff
@@ -0,0 +1,14 @@
+// Automatically generated - do not modify!
+
+package react.dom
+
+@Suppress("NAME_CONTAINS_ILLEGAL_CHARS")
+// language=JavaScript
+@JsName("""({mouse: 'mouse', pen: 'pen', touch: 'touch'})""")
+external enum class PointerType {
+    mouse,
+    pen,
+    touch,
+
+    ;
```

Comment:
Why do you use semicolon, if there is trailing coma?

Reply:
It's delimiter for constants and members. New members will not produce delimiters